packages/@okta/vuepress-theme-prose/const/navbar.const.js

```diff
@@ -461,6 +461,166 @@ export const languagesSdk = [
+          { title: "Dynamic Client Registration", path: "/docs/reference/api/oauth-clients/"},
+          { title: "Event Hooks", path: "/docs/reference/api/event-hooks/" },
+          { title: "Event Types", path: "/docs/reference/api/event-types/" },
+          { title: "Factors", path: "/docs/reference/api/factors/" },
+          { title: "Features", path: "/docs/reference/api/features/" },
+          { title: "Groups", path: "/docs/reference/api/groups/" },
+          { title: "Inline Hooks", path: "/docs/reference/api/inline-hooks/" },
+          { title: "Linked Objects", path: "/docs/reference/api/linked-objects/"},
+          { title: "Mappings", path: "/docs/reference/api/mappings/" },
+          { title: "MyAccount", path: "/docs/reference/api/myaccount/" },
+          { title: "Org", path: "/docs/reference/api/org/" },
+          { title: "Policy", path: "/docs/reference/api/policy/" },
+          { title: "Schemas", path: "/docs/reference/api/schemas/" },
+          { title: "Sessions", path: "/docs/reference/api/sessions/" },
+          { title: "System Log", path: "/docs/reference/api/system-log/" },
+          { title: "Templates", path: "/docs/reference/api/templates/" },
+          { title: "ThreatInsight", path: "/docs/reference/api/threat-insight/" },
+          { title: "Trusted Origins", path: "/docs/reference/api/trusted-origins/" },
+          { title: "User Types", path: "/docs/reference/api/user-types/" },
+          { title: "Users", path: "/docs/reference/api/users/" },
+          { title: "Zones", path: "/docs/reference/api/zones/" }
+        ]
+      },
+      {
+        title: 'Advanced Server Access Api',
+        subLinks: [
+           {
+            title: "Introduction to the Advanced Server Access API",
+            path: "/docs/reference/api/asa/introduction/"
+          },
+          {
+            title: "ASA Attributes API",
+            path: "/docs/reference/api/asa/attributes/"
+          },
+          { title: "ASA Audits API", path: "/docs/reference/api/asa/audits/" },
+          {
+            title: "ASA Clients API",
+            path: "/docs/reference/api/asa/clients/"
+          },
+          {
+            title: "ASA Entitlements API",
+            path: "/docs/reference/api/asa/entitlements/"
+          },
+          { title: "ASA Groups API", path: "/docs/reference/api/asa/groups/" },
+          {
+            title: "ASA Projects API",
+            path: "/docs/reference/api/asa/projects/"
+          },
+          {
+            title: "ASA Service Users API",
+            path: "/docs/reference/api/asa/service-users/"
+          },
+          { title: "ASA Teams API", path: "/docs/reference/api/asa/teams/" },
+          { title: "ASA Users API", path: "/docs/reference/api/asa/users/" }
+        ]
+      },
+      {
+        title: 'SCIM Protocol API',
+        subLinks: [
+          { 
+            title: "SCIM overview",
```

Comment:
~SCIM~ **Overview**
(No need to have SCIM in this title, as it's not in the linked file and it's clear that it has to do with SCIM because of the top level heading).